### DISCUSSION TITLE
error: Error: Unable to resolve module ./assets/toggle-drawer-icon.png

### DISCUSSION THREAD

**Discussion by Tosinkoa** [react-navigation/react-navigation#11591] [Q&A]
After a successful installation of @react-navigation/drawer package, I got an error that `asset` folder is missing in the node_module, it turns out that this folder is not getting attached during installation (i.e while `yarn add @react-navigation/drawer` is running). I have checked the repo for the package and the folder is there [https://github.com/react-navigation/react-navigation/tree/main/packages/drawer/src/views](https://github.com/react-navigation/react-navigation/tree/main/packages/drawer/src/views). Pls any idea on how to fix this?

error: Error: Unable to resolve module ./assets/toggle-drawer-icon.png from C:\Users\Paul\Desktop\My-Codes\AwesomeProject\node_modules\@react-navigation\drawer\src\views\DrawerToggleButton.tsx:

None of these files exist:
  * toggle-drawer-icon.png
  * node_modules\@react-navigation\drawer\src\views\assets\toggle-drawer-icon.png\index(.native|.android.js|.native.js|.js|.android.jsx|.native.jsx|.jsx|.android.json|.native.json|.json|.android.ts|.native.ts|.ts|.android.tsx|.native.tsx|.tsx)
  35 |       <Image
  36 |         style={[styles.icon, tintColor ? { tintColor } : null]}
> 37 |         source={require('./assets/toggle-drawer-icon.png')}
     |                          ^
  38 |         fadeDuration={0}
  39 |       />
  40 |     </PlatformPressable>
![image](https://github.com/react-navigation/react-navigation/assets/90416105/8ae88a7c-ca23-443c-83b2-af16526c4451)
![image](https://github.com/react-navigation/react-navigation/assets/90416105/60253289-67cc-4d01-a201-52feab0388fc)

Edit:
I have followed the troubleshooting step for `Unable to resolve module` in the doc, but this issue still persist. [here](https://reactnavigation.org/docs/troubleshooting#im-getting-an-error-unable-to-resolve-module-after-updating-to-the-latest-version)

**Comment by mdpmultiplica** [comment]
Hi @Tosinkoa Did you manage to solve this problem? I'm facing the same issue here

**Reply by Tosinkoa** [reply]
Yes I resolved it by installing `postinstall` package and added it to my `package.json` script.
Like this:
```
  "scripts": {
    ...
    "postinstall": "postinstall",
  },
  ```
  
  Here is the package: https://www.npmjs.com/package/postinstall
  
  Edit: After installation, run `yarn` command
